asreview/data/statistics.py

```diff
@@ -0,0 +1,203 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+
+def n_records(data):
+    """Return the number of records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    return len(data)
+
+
+def n_included(data):
+    """Return the number of included records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    if data.labels is not None:
+        return len(np.where(data.labels == 1)[0])
+    return None
+
+
+def n_excluded(data):
+    """Return the number of excluded records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    if data.labels is None:
+        return None
+    return len(np.where(data.labels == 0)[0])
+
+
+def n_unlabeled(data):
+    """Return the number of unlabeled records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    if data.labels is None:
+        return None
+    return len(data.labels) - n_included(data) - n_excluded(data)
+
+
+def n_missing_title(data):
+    """Return the number of records with missing titles.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    n_missing = 0
+    if data.title is None:
+        return None, None
+    if data.labels is None:
+        n_missing_included = None
+    else:
+        n_missing_included = 0
+    for i in range(len(data.title)):
+        if len(data.title[i]) == 0:
+            n_missing += 1
+            if (data.labels is not None and data.labels[i] == 1):
+                n_missing_included += 1
+    return n_missing, n_missing_included
+
+
+def n_missing_abstract(data):
+    """Return the number of records with missing abstracts.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    n_missing = 0
+    if data.abstract is None:
+        return None, None
+    if data.labels is None:
+        n_missing_included = None
+    else:
+        n_missing_included = 0
+
+    for i in range(len(data.abstract)):
+        if len(data.abstract[i]) == 0:
```

Comment:
Should we really use zero, or maybe less than an x-number of characters?
@bmkramer what do you think?

Reply:
I would say missing should be reserved for len = 0 (or title = NA, depending on how it's in the dataset)

Any other decision on which length to discard is subjective and should be documented as such, but as a separate parameter imo

Reply:
Thanks. Sounds like a preprocessing step for the user.